server/app/Main.hs

```diff
@@ -1,4 +1,46 @@
 import Lib
+import Options (Option (..), runOptionParser)
+import qualified Options.Applicative as Opt
+
+getConfig :: IO ServerConfig
+getConfig = runOptionParser $ \envOption ->
+  ServerConfig
+    <$> envOption
+      ( Option
+          { optHelp = "Path to the nix program database.",
+            optMetaVar = "PATH",
+            optLong = "program-db",
+            optShort = 'd',
+            optEnvKey = "NIX_PROGRAM_DB",
+            optReadM = ProgramDB <$> Opt.str,
+            optDefault = ProgramDB "/nix/var/nix/profiles/per-user/root/channels/nixos/programs.sqlite",
+            optShow = unProgramDB
+          }
+      )
+    <*> envOption
+      ( Option
+          { optHelp = "Path to the application database.",
+            optMetaVar = "PATH",
+            optLong = "app-db",
+            optShort = 'a',
+            optEnvKey = "APP_DB",
```

Comment:
`BINPLZ_APPLICATION_DB` or `BINPLZ_APP_DB`.

I prefer long name here. This is going to be an interface to the outer space and the scope is very large.
And normally, we don't handwrite the name env vars often.